### DISCUSSION TITLE
limitations on the number of messages displayed?

### DISCUSSION THREAD

**Discussion by Gurulam** [Pustur/whatsapp-chat-parser-website#35] [Q&A] [answered]
I congratulate you for the initiative to create this project that solves to a great extent the problems of displaying whatsapp chats after being exported. My question is the following, I have loaded the compressed file exported from whatsapp, which corresponds to a group chat including media resources, however, when viewing on the website only shows me a limited number of messages as well as photos and media resources, for example, of a total of 200 messages only shows me 24 and I would like to know if the version of the website is limited to a maximum number of messages or is it because they correspond to group messages?  Can I use the local installation version to be able to see all the messages contained in the txt file including images?
Thank you for your help

**Comment by Pustur** [comment, marked as answer]
Hi @Gurulam 

The website (locally or otherwise, they're the same) by default limits the visible messages to reduce lag in chats with many messages, however you can control how many messages you wish to see using the filter located in the hamburger menu at the bottom left (visible only once the chat has been loaded)

If you still can't see all messages it may be an issue of the parser, in that case I would have to ask you to send me the chat so I can see what's going on. You can send it to the email listed on my profile.

**Reply by Gurulam** [reply]
Excellent!, thank you very much @Pustur . I tested it again and it is correct. 
best regards.
